### PR TITLE
Updating components elevations

### PR DESCRIPTION
**Pull request by mfferreira**

### CODE REVIEW

File: src/stylus/components/_pagination.styl [hunk 1]
```diff
@@ -37,12 +37,12 @@
     text-decoration: none
     
     &--active
-      @extend .z-depth-2
+      elevation(4)
```
Comment (dohomi) [inline src/stylus/components/_pagination.styl]: @mfferreira I really like the elevation on active. shall we maybe introduce the same elevation(4) to hover?

File: src/stylus/components/_sidebar.styl [hunk 2]
```diff
@@ -7,7 +7,7 @@
   width: $sidebar-width
   max-width: 100%
   z-index: 3
-  @extend .z-depth-2
+  elevation(16)
```
Comment (dohomi) [inline src/stylus/components/_sidebar.styl]: Is elevetion 16 based on material-specs? I think its little bit too much. I think elevation(8) if is more suitable. I use a 4K monitor and it just looks bit off.

**Comment by johnleider** [comment]
Need final approval from rest of members @dohomi @prograhammer

**Comment by dohomi** [comment]
@johnleider I added 2 comments but they are both minor things, approved..

**Comment by johnleider** [comment]
David is probably somewhere in the jungle, will give until the end of today for his approval.

**Pull request merged**